otsdaq-mu2e/FEInterfaces/DTCFrontEndInterfaceImpl.cc

```diff
@@ -1553,6 +1572,13 @@ void DTCFrontEndInterface::configureHardwareDevMode(void)
 	thisDTC_->SetCFOEmulationNumNullHeartbeats(0x10);
 	// registerWrite(0x91AC, 0); // Set number of heartbeats packets
 
+    // If this is a CRV ROC, enable the punched clock by default
+	__FE_COUT__ << "firmware flavour is " << getCFOandDTCRegisters()->ReadDesigFlavour() << __E__;
+	if(getCFOandDTCRegisters()->ReadDesigFlavour() == "e")
```

Comment:
I think we should trade "ReadDesignFlavour" for something like "isCRVDTCDesignFlavour()," "isNonCRVDTCDesignFlavour()," and "isCFODesignFlavour()" which each just returns true/false. This way the implementation is contained within getCFOandDTCRegisters and the interpretation is simple by only using booleans.

Reply:
done